lib/wibox/container/only_on_screen.lua

```diff
@@ -0,0 +1,145 @@
+---------------------------------------------------------------------------
+--
+-- @author Uli Schlachter
+-- @copyright 2017 Uli Schlachter
+-- @classmod wibox.container.only_on_screen
```

Comment:
> "wibox is independent from Awesome"

Huh? Wibox is based on drawin which is part of awesome. It might have low number of interdependences, but it is still there.